ceremonies/2023/run.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash -e
+
+set -o pipefail
+
+function usage() {
+    echo -e "Usage:
+
+    ./$(basename "${0}")
+    "
+}
+
+if [ "${1}" == "-h" ]; then
+    usage
+    exit 0
+fi
+
+function _echo() {
+    echo "$(date +'%Y/%m/%d %H:%M:%S') ${@}"
+}
+
+function setup_ceremony_tool() {
+    if [ -n "${CEREMONY_BIN_2023}" ] && [ -x "${CEREMONY_BIN_2023}" ]; then
+        return 0
+    fi
+
+    TMPDIR="/tmp/ceremony-tools"
+    export CEREMONY_BIN_2023="${TMPDIR}/bin/2023/ceremony"
+    mkdir -p "${TMPDIR}/bin/2023/"
+    if [ ! -d "${TMPDIR}/boulder" ]; then
+        git clone https://github.com/letsencrypt/boulder/ "${TMPDIR}/boulder"
+    fi
+
+    if [ ! -x "${TMPDIR}/bin/2023/ceremony" ]; then
+        # Build ceremony on the commit prior to removing configuration of Policy OIDs.
+        # This will allow all ceremonies prior to 2023 to complete successfully without
+        # requiring backporting changes to those ceremonies and losing the historical
+        # representation of the ceremony.
+        cd "${TMPDIR}/boulder"
+        git checkout 72e01b337abccc9b849a3063666943489bcc573d
+        make
+        cd -
+        cp "${TMPDIR}/boulder/bin/ceremony" "${TMPDIR}/bin/2023/"
+    fi
```

Comment:
This _is_ the 2023 ceremony -- I don't think we need the special handling for checking out an old version anymore in this one.

Reply:
Gotcha, the person who adds the next ceremony can populate this when needed.

Reply:
Gotcha, so when the next ceremony happens, and if it happens to change functionality that affects the historic 2023 ceremony, this file can be changed to check out a specific commit.